pkg/kubelet/nodestatus/setters.go

```diff
@@ -792,7 +792,7 @@ func VolumeLimits(volumePluginListFunc func() []volume.VolumePluginWithAttachLim
 		for _, volumePlugin := range pluginWithLimits {
 			attachLimits, err := volumePlugin.GetVolumeLimits()
 			if err != nil {
-				klog.V(4).InfoS("Error getting volume limit for plugin", "plugin", volumePlugin.GetPluginName())
+				klog.V(4).InfoS("skipping volume limits for volume plugin", "plugin", volumePlugin.GetPluginName())
```

Comment:
```suggestion
				klog.V(4).InfoS("Skipping volume limits for volume plugin", "plugin", volumePlugin.GetPluginName())
```

Reply:
@yangjunmyfm192085 thanks for your review 🙏🏽